ext/js/display/display-anki.js

```diff
@@ -1139,6 +1149,32 @@ export class DisplayAnki {
         return viewNoteButton;
     }
 
+    /**
+     * @param {(?import('anki').NoteInfo)[]} noteInfos
+     * @param {HTMLButtonElement} viewNoteButton
+     * @returns {HTMLButtonElement}
+     */
+    _setViewNoteButtonCardState(noteInfos, viewNoteButton) {
+        if (this._isAdditionalInfoEnabled() === false) { return viewNoteButton; }
```

Comment:
While we know it will be true in this case, there no guarantee that `this._isAdditionalInfoEnabled()` being false will always mean `noteInfos` is empty. Just check `noteInfos` here.

Reply:
It being false means these feature isn't enable

Reply:
I'm asking for this change not because the code doesn't work but because it could be messier in the future.

For example, if there is a change later that has a case where `this._isAdditionalInfoEnabled()` evaluates true but `noteInfos` is empty. You are checking whether things *should* be correct not whether they actually are correct.

How about this:

```suggestion
        if (this._isAdditionalInfoEnabled() === false || noteInfos.length === 0) { return viewNoteButton; }
```

Reply:
> I'm asking for this change not because the code doesn't work but because it could be messier in the future.
> 
> For example, if there is a change later that has a case where `this._isAdditionalInfoEnabled()` evaluates true but `noteInfos` is empty. You are checking whether things _should_ be correct not whether they actually are correct.
> 
> How about this:

done